relayer/crates/starknet-cli/examples/bootstrap_ibc.rs

```diff
@@ -297,6 +297,16 @@ fn main() -> Result<(), Error> {
             );
         }
 
+        info!(
+            "Osmosis: client {}, connection {}, channel {}",
+            cosmos_client_id, cosmos_connection_id, cosmos_channel_id
+        );
+
+        info!(
+            "Starknet: client {}, connection {}, channel {}",
+            starknet_client_id, starknet_connection_id, starknet_channel_id
+        );
+
```

Comment:
@soareschen any idea why the `info!` logs in this file doesn't actually print anything? maybe the logger is not properly initialized, as this is not really a `#[cfg(test)]`.

Reply:
Probably because by default we filter out all targets don't start with "hermes". You can add `target: "hermes",` inside `info!` to make it pass the filter.

The filter is needed as otherwise going to `DEBUG` or lower will show a lot of logs generated from third party libraries.

Reply:
64186eb